admin/class-yoast-dashboard-widget.php

```diff
@@ -28,12 +28,8 @@ class Yoast_Dashboard_Widget {
 	/**
 	 * @param WPSEO_Statistics $statistics The statistics class to retrieve statistics from.
 	 */
-	public function __construct( WPSEO_Statistics $statistics = null ) {
-		if ( null === $statistics ) {
-			$statistics = new WPSEO_Statistics();
-		}
-
-		$this->statistics    = $statistics;
+	public function register_hooks() {
```

Comment:
You removed the constructor. That is not desirable, because this is a breaking this.

Reply:
You also removed the possibility of injecting the dependency. I would love to see the constructor where you use the old way for injecting the dependency.